src/orquestra/sdk/_base/_dsl.py

```diff
@@ -493,6 +490,14 @@ def __call__(self, *args: _P.args, **kwargs: _P.kwargs) -> _R:
                 f"def {self._fn_name}(): ..."
             )
             raise InvalidTaskDefinitionError(err)
+
+    def validate_task(self):
+        self._validate_task_not_in_main()
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏As this is in the public API, can you add a docstring?